### PR TITLE
use brace expansion for computed property

### PR DESCRIPTION
**Pull request by Daniel-Xu**

### CODE REVIEW

File: engineering/ember.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default Component.extend({
   post: alias('myPost'),
 
   // Multiline CP
-  authorName: computed('author.firstName', 'author.lastName', {
+  authorName: computed('author.{firstName,lastName}', {
```
Comment (bcardarella) [inline engineering/ember.md]: space after `,`

Comment (bcardarella) [inline engineering/ember.md]: more specific: `firstName, lastName`

Comment (rsocci) [inline engineering/ember.md]: @bcardarella they must be together (without space) for the brace expansion to work

**Comment by hbrysiewicz** [comment]
Would have said space but apparently not... looks good

**Pull request merged**